packages/compiler/core/library.ts

```diff
@@ -1,6 +1,14 @@
 import { createDiagnosticCreator } from "./diagnostics.js";
 import { CadlLibrary, CadlLibraryDef, CallableMessage, DiagnosticMessages } from "./types.js";
 
+const globalLibraryUrlLoadedSym = Symbol.for("CADL_LIBRARY_URL_LOADED");
+(globalThis as any)[globalLibraryUrlLoadedSym] = new Set<string>();
```

Comment:
wouldn't this always overwrite an existing global? Maybe we should check if one exists before reinitializing it?

Reply:
hhm yes... but somehow the test were working